CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+
+## v3.1.1 (unreleased)
```

Comment:
3.2?

Reply:
This feature is for internal use only, I don't want to announce it! :)